packages/mobile/src/navigator/QRNavigator.tsx

```diff
@@ -112,7 +112,7 @@ function AnimatedScannerScene({ route, position, ...props }: AnimatedScannerScen
   )
 
   return (
-    <Animated.View style={animatedStyle}>
+    <Animated.View style={[animatedStyle]}>
```

Comment:
Looks like this is just silencing a TS error?

Reply:
Without the array it wasn't building. I do see use memo used for animated styles in other places so I am not sure why this started failing with the update.

Reply:
ok then can you change it to? This way it's a bit more clear there's a TS issue.
```suggestion
    // @ts-expect-error
    <Animated.View style={animatedStyle}>
```

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2230/commits/3786a5a828713cd1827e313eb9cde92c14ee9f40!